packages.ini

```diff
@@ -206,6 +206,8 @@ validate_incorrect_missing_deps = six
 [flask==2.1.2]
 [flask==2.2.2]
 
+[flask-redis==0.3.0]
```

Comment:
can you also add the latest (flask-redis==0.4.0)?

Reply:
absolutely.